src/components/LicenseBox.jsx

```diff
@@ -127,31 +125,89 @@ export const ImageContent = ({ t, image }) => {
         <h2>{t('license.images.heading')}</h2>
         <p>{t('license.images.description')}</p>
       </div>
+      <MediaList>
+        {images.map(image => (
+          <MediaListItem key={image.src}>
+            <MediaListItemImage>
+              <img src={image.src} alt={image.altText} />
+            </MediaListItemImage>
+            <MediaListItemBody
+              license={image.copyright.license.license}
+              title={t('license.images.rules')}
+              resourceUrl=""
+              locale="nb"
+              resourceType="image">
+              <MediaListItemActions>
+                <div className="c-medialist__ref">
+                  <MediaListItemMeta items={getLicenseItems(image, t)} />
+                  <CopyTextButton
+                    hasCopiedTitle={t('license.hasCopiedTitle')}
+                    copyTitle={t('license.copyTitle')}
+                    stringToCopy={getCopyrightCopyString(image.copyright, t)}
```

Comment:
Her kan du bruke copyText som nettopp blei innført. Men då må du evt sende inn path som parameter til grapgql.

Reply:
Kan gjøre det for VisualElementContent, men for Text og Image blir det vanskeligere for der bruker vi ikke article-converter